src/dodal/plans/check_topup.py

```diff
@@ -23,15 +39,28 @@ def _gating_permitted(machine_mode: SynchrotronMode):
     return False
 
 
-def _delay_to_avoid_topup(total_run_time, time_to_topup):
+def _delay_to_avoid_topup(
```

Comment:
Could: We could have a docstring on this function explaining why we need the threshold and why sometimes wait and sometimes don't (basically just summarising this issue).